doco/mdsource/query-usage.source.md

```diff
@@ -70,7 +69,6 @@ All where statements require a `path`. This is a full path to a, possible nested
  * `startsWith`: Only works with `string`
  * `endsWith`: Only works with `string`
  * `in`: Check if a member existing in a given collection of values
- * `notIn`: Check if a member doesn't exist in a given collection of values
```

Comment:
was notin removed?

Reply:
Yeah it was, removed it since the negate option was added to the where expression.

Bit of a short sighted decision on my part considering this removes backwards compatibility to all the existing user's queries.

I can re-add it.

Reply:
Re-added the notIn operator and verified it working